save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphCollectionView.kt

```diff
@@ -97,46 +100,83 @@ val fossGraphCollectionView: FC<FossGraphCollectionViewProps> = FC { props ->
-            }
-        }
-    }
-
-    div {
-        @Suppress("TOO_MANY_LINES_IN_LAMBDA")
-        props.currentUserInfo?.globalRole?.let { role ->
-            val tabList = if (role.isHigherOrEqualThan(Role.SUPER_ADMIN)) {
-                VulnerabilityListTab.values().map { it.name }
-            } else {
-                VulnerabilityListTab.values().filter { it != VulnerabilityListTab.ADMIN }.map { it.name }
-            }
-            tab(selectedMenu.name, tabList, "nav nav-tabs mt-3") { value ->
-                setSelectedMenu { VulnerabilityListTab.valueOf(value) }
-                setVulnerabilityFilters {
-                    when (VulnerabilityListTab.valueOf(value)) {
-                        VulnerabilityListTab.PUBLIC -> VulnerabilityFilter(prefixName = "", active = true)
-                        VulnerabilityListTab.ADMIN -> VulnerabilityFilter(prefixName = "", active = false)
-                        VulnerabilityListTab.OWNER -> VulnerabilityFilter(prefixName = "", active = false, isOwner = true)
+    main {
+        className = ClassName("main-content mt-0 ps")
+        div {
+            className = ClassName("page-header align-items-start min-vh-100")
+            div {
+                className = ClassName("d-flex justify-content-center")
+                div {
+                    className = ClassName("col-md-4 d-flex align-items-stretch")
+                    div {
+                        div {
+                            className = ClassName("mb-2")
+                            fossGraphIntroductionComponent()
+                        }
                     }
                 }
-            }
-        }
+                div {
+                    className = ClassName("card flex-md-row col-lg-7 d-flex align-items-stretch")
+                    div {
+                        className = ClassName("col")
+                        div {
+                            className = ClassName("d-flex justify-content-center mb-1 mt-2")
+                            withNavigate { navigateContext ->
+                                buttonBuilder(label = "Propose a vulnerability", style = "info") {
+                                    navigateContext.navigate("/${FrontendRoutes.CREATE_VULNERABILITY}")
+                                }
+                            }
+                        }
 
-        fossGraphTable {
-            filters = vulnerabilityFilters
-            getData = { _, _ ->
-                post(
-                    url = "$apiUrl/vulnerabilities/by-filters",
-                    headers = jsonHeaders,
-                    body = Json.encodeToString(vulnerabilityFilters),
-                    loadingHandler = ::noopLoadingHandler,
-                    responseHandler = ::noopResponseHandler,
-                ).unsafeMap {
-                    it.decodeFromJsonString()
+                        div {
+                            @Suppress("TOO_MANY_LINES_IN_LAMBDA")
```

Comment:
can we configure this rule somehow to not to trigger on frontend?

Reply:
We can only adjust the maximum size of the lamda itself.
We can create an issue on Diktat.